content/docs/started/_index.md

```diff
@@ -3,3 +3,25 @@ title = "Getting Started"
 description = "How to get started using Kubeflow."
 weight = 20
 +++
+
+The guides in this section give detailed information about using Kubeflow and 
+its components.
+
+For best understanding of the guides, it's useful to have some knowledge of
+the following systems:
+
+* [Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+* [TensorFlow](https://www.tensorflow.org/get_started/)
+* [ksonnet](https://ksonnet.io/docs/tutorial)
+
+##### Kubeflow requirements
+
+ * ksonnet version {{% ksonnet-min-version %}} or later. See the 
+   [ksonnet component guide](/docs/components/ksonnet/) for details about
+   installing ksonnet.
+ * An existing Kubernetes cluster using Kubernetes version 
+   {{% kubernetes-min-version %}} or later:
+
+   * A minimum of 0.6 CPU in cluster (Reserved for 3 replicated ambassador pods and according to your need add additional CPUs)
```

Comment:
Do these general requirements hold for most use cases?

Reply:
Same as previous comment. I'm not sure, but I wanted to focus on restructuring existing content rather than verifying its accuracy